src/background_page/modes.js

```diff
@@ -57,12 +57,10 @@ export async function reloadAllClients () {
   console.log(tabs)
   try {
     await Promise.all(
-      tabs.filter(isTabExecutable).map(tab =>
-        browser.tabs.executeScript(tab.id, {
-          file: 'content_script_loader.js',
-          allFrames: true,
-          runAt: 'document_start',
-          matchAboutBlank: true
+      tabs.map(tab =>
+        await chrome.scripting.executeScript({
```

Comment:
Hey @ceuk, on running your version of this plugin I ran into an error.

```
ERROR in ./src/background_page/modes.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /home/adamryman/projects/github.com/lusakasa/saka-key/src/background_page/modes.js: Can not use keyword 'await' outside an async function (61:8)

  59 |     await Promise.all(
  60 |       tabs.map(tab =>
> 61 |         await chrome.scripting.executeScript({
     |         ^
  62 |           target: { tabId: tab.id, allFrames: true },
  63 |           files: ['content_script_loader.js'],
  64 |         })
    at Object.raise (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:7017:17)
    at Object.checkReservedWord (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10401:14)
    at Object.parseIdentifierName (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10363:12)
    at Object.parseIdentifier (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10335:23)
    at Object.parseExprAtom (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9516:27)
    at Object.parseExprAtom (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:4120:20)
    at Object.parseExprSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9259:23)
    at Object.parseMaybeUnary (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9239:21)
    at Object.parseExprOps (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9109:23)
    at Object.parseMaybeConditional (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9082:23)
    at Object.parseMaybeAssign (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9037:21)
    at Object.parseFunctionBody (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10234:24)
    at Object.parseArrowExpression (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10209:10)
    at Object.parseExprAtom (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9547:18)
    at Object.parseExprAtom (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:4120:20)
    at Object.parseExprSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9259:23)
    at Object.parseMaybeUnary (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9239:21)
    at Object.parseExprOps (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9109:23)
    at Object.parseMaybeConditional (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9082:23)
    at Object.parseMaybeAssign (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9037:21)
    at Object.parseExprListItem (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10327:18)
    at Object.parseCallExpressionArguments (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9440:22)
    at Object.parseSubscript (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9346:31)
    at Object.parseSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9276:19)
    at Object.parseExprSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9265:17)
    at Object.parseMaybeUnary (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9239:21)
    at Object.parseExprOps (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9109:23)
    at Object.parseMaybeConditional (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9082:23)
    at Object.parseMaybeAssign (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9037:21)
    at Object.parseExprListItem (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10327:18)
    at Object.parseCallExpressionArguments (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9440:22)
    at Object.parseSubscript (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9346:31)
    at Object.parseSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9276:19)
    at Object.parseExprSubscripts (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9265:17)
    at Object.parseMaybeUnary (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:9239:21)
    at Object.parseAwait (/home/adamryman/projects/github.com/lusakasa/saka-key/node_modules/@babel/parser/lib/index.js:10442:28)
 @ ./src/background_page/index.js 3:0-84 13:2-12 14:2-15 21:0-9 25:0-5
```

Unfortunately, I don't have much experience with the `Promise`/`await` styles of writing javascript. 

If anyone could take a look, that would be great :)

Reply:
OMG, just removing the await seemed to get us past that error.